rego/rules/arm/authorization/custom_owner_role.rego

```diff
@@ -35,22 +36,35 @@ __rego__metadoc__ := {
 
 input_type = "arm"
 
-resource_type = "MULTIPLE"
+resource_type = "Microsoft.Authorization/roledefinitions"
 
-resources = fugue.resources("Microsoft.Authorization/roleDefinitions")
+is_subscription_scope(scope) {
+	scope == "/"
+}
+
+# Examples:
+# * "/subscriptions/479a226b-4153-48f7-8943-3e8e388a93cb"
+# * "/subscriptions/479a226b-4153-48f7-8943-3e8e388a93cb/"
+is_subscription_scope(scope) {
+	re_match(`^/subscriptions/[^/]+/?$`, lower(scope))
+}
 
-is_invalid(resource) {
-	resource.TODO == "TODO" # FIXME
+# Examples:
+# * "[concat('/subscriptions/', subscription().subscriptionId)]"
+# * "[concat('/subscriptions/', '479a226b-4153-48f7-8943-3e8e388a93cb')]"
+# * "[concat('/subscriptions/', parameters('subscriptionId'))]"
+is_subscription_scope(scope) {
+	re_match(`^\[concat\('/subscriptions/',[^,]+\]$`, replace(lower(scope), " ", ""))
```

Comment:
I'm hoping once we see a few of these patterns we can come up with some utilities that parse `concat` and other calls into easier to match arrays...

Reply:
That's a good point. These can get pretty complex, so it might be better to do that in the loader.